### PR TITLE
Setup Application Environments

### LINKED ISSUE #8: [3] Setup the different application environments
It is important that we have different environments for development, testing and development. In that regard;
- Configure the different application environments i.e development, testing, production.
- Ensure the application runs from the `run.py` file.

### PR DESCRIPTION
**Pull request by iTaylorCodes**
### What does this PR do?
- This Pull Request sets up development, testing, and production environments.
- This PR closes #8 
### Description of Task to be completed
- Create config.py file to control each environment. Includes environment classes, TestingConfig, DevelopmentConfig, ProductionConfig. Change FLASK_APP to run application from run.py. Add logic to run.py to control which environment to use depending on FLASK_ENV.
### How should this be manually tested?
- From the CLI use "export FLASK_ENV=testing", "export FLASK_ENV=production", or "export FLASK_ENV=development" to change to each environment respectively. Then use "flask run" to start the application in that environment.

### CODE REVIEW

**Review by alexkayabula — CHANGES_REQUESTED** [review]
@iTaylorCodes  read through this resource and set up the environments accordingly https://flask.palletsprojects.com/en/2.0.x/config/#development-production.

File: Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web: FLASK_APP=app.py python -m flask run --host=0.0.0.0 --port=$PORT
```
Comment (alexkayabula) [inline Procfile]: @iTaylorCodes what is the use of this set up?

File: runtime.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+python-3.8.11
```
Comment (alexkayabula) [inline runtime.txt]: @iTaylorCodes what is the use of this setup ?

**Review by alexkayabula — CHANGES_REQUESTED** [review]

File: Procfile (hunk 1 above)
Comment (iTaylorCodes) [reply Procfile]: This Procfile was for Heroku deployment, but I was creating the environments incorrectly. Removed file.

File: runtime.txt (hunk 2 above)
Comment (iTaylorCodes) [reply runtime.txt]: This runtime.txt file was to tell Heroku which version of python to use. Removed file.

**Review by alexkayabula — CHANGES_REQUESTED** [review]
@iTaylorCodes the description of this PR is not clear.
1. `Added logic to app.py to control which environment to be in depending on FLASK_ENV.`
     - I didn't see any changes in the app.y
2. `Use print(app.config['ENV']) to verify our app environment matches the FLASK_ENV`
     - I would rather you indicate how to run the application in the development environment or testing environment.

**Review by alexkayabula — COMMENTED** [review]
@iTaylorCodes please let me know how I can run the application with this new configuration.

**Comment by iTaylorCodes** [comment]
Added, "Then use 'flask run' to start the application in that environment.", to the "How should this be manually tested?" section of the PR description.

**Review by alexkayabula — CHANGES_REQUESTED** [review]
@iTaylorCodes the application does not run. Please look into this issue and add screenshots if necessary.

**Comment by iTaylorCodes** [comment]
Here's the application running in each environment:
Screen Shot 2021-07-24 at 3 07 28 PM
Giving a 404 page because there are no views setup yet.

**Review by alexkayabula — APPROVED** [review]
LGTM

**Pull request merged**